collect_app/src/main/java/org/odk/collect/android/views/ODKView.java

```diff
@@ -483,7 +484,20 @@ public void highlightWidget(FormIndex formIndex) {
                 scrollTo(0, qw.getTop());
 
                 ValueAnimator va = new ValueAnimator();
-                va.setIntValues(getResources().getColor(R.color.red), getDrawingCacheBackgroundColor());
+                if (Build.VERSION.SDK_INT > Build.VERSION_CODES.KITKAT) {
+                    // only for kitkat and newer versions
+                    va.setIntValues(getResources().getColor(R.color.red), getDrawingCacheBackgroundColor());
+                    } else {
```

Comment:
Please fix the indentation here.

Reply:
I don't know, why it's showing it like this here, it's completely okay on my laptop.